node/router/src/handshake.rs

```diff
@@ -318,7 +318,7 @@ impl<N: Network> Router<N> {
         if self.is_connected(&peer_ip) {
             bail!("Dropping connection request from '{peer_ip}' (already connected)")
         }
-        // Only allow trusted peers to connect if allow_external_peers is set
```

Comment:
Thanks for catching this. I might have written this and obviously meant *untrusted peers*.